src/modules/shopgate/helpers/basket.php

```diff
@@ -204,6 +206,20 @@ protected function loadOrderBasketPayment(shopgate_oxbasket &$oxBasket, Shopgate
     }
 
     /**
+     * @param shopgate_oxbasket $oxBasket
+     * @param ShopgateCartBase  $shopgateOrder
```

Comment:
You should add "@post will overwrite the total price of the passed $oxidBasket" or something similar.

Thats because someone who calls the method needs to know what effects the method will cause